srcpkgs/dua-cli/patches/0001-PATCH-fix-on-platforms-without-64-bit-atomics.patch

```diff
@@ -0,0 +1,31 @@
+From e49a4bfc495e347834f7b3e8fc3a1501a0b313d7 Mon Sep 17 00:00:00 2001
```

Comment:
Could you rename the file and remove the header differences to minimize the patch?

If you think they could take this upstream too, that'd be great!